app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb

```diff
@@ -49,8 +49,13 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager < ManageI
   supports :catalog
   supports :create
   supports :provisioning
+  supports :native_console
```

Comment:
Can you move this above `supports :provisioning` to keep this list alphabetical?

Reply:
sure Adam. I had it down because of the length of native_console is biggest among all of them :-).